zilencer/management/commands/populate_db.py

```diff
@@ -155,7 +156,8 @@ def handle(self, **options):
                 for type_id in recipient_streams[:int(len(recipient_streams) *
                                                       float(i)/len(profiles)) + 1]:
                     r = Recipient.objects.get(type=Recipient.STREAM, type_id=type_id)
-                    s = Subscription(recipient=r, user_profile=profile)
+                    s = Subscription(recipient=r, user_profile=profile, color="#%06x" % random.randint(0, 0xFFFFFF))
+
```

Comment:
Cool, I think it'd be good to make the stream color selection more deterministic and consistent with what would happen if you actually subscribed to the streams.  The basic story for how to do that is that we should reuse the `stream_assignment_colors` structure defined in `zerver/lib/actions.py`, and just select streams in order.

E.g. `color=stream_assignment_colors[j]` where this is the `j`th Subscription object made for that user in this function.

(and we can use the same algorithm where we set stream colors below as well)